npm-shrinkwrap.json

```diff
@@ -1998,7 +1995,7 @@
       }
     },
     "jsdom": {
-      "version": "9.2.1",
+      "version": "7.2.2",
```

Comment:
We probably should bump version number in https://github.com/Automattic/react-test-env.